### DISCUSSION TITLE
download

### DISCUSSION THREAD

**Discussion by AmatorVeritatis** [esp8266/Arduino#9088] [Q&A]
Hello Guys! I am a beginner. So sorry for my stupid question. How can I download a library? I am trying to download this https://github.com/esp8266/Arduino/tree/master/libraries/ESP8266WiFi and I can't see a button download...

**Comment by d-a-v** [comment]
This library is included with the esp8266 Arduino core that you have installed in the Arduino IDE (also true with platformIO/vscode, eclipse, ...).
If you don't see it, it may be because you did not select the right core before compiling, otherwise it is an installation error.
Be sure to follow [instructions](https://arduino-esp8266.readthedocs.io/en/latest/installing.html).

**Comment by AmatorVeritatis** [comment]
Many thanks for your answer...
